### DISCUSSION TITLE
运行时间长后，自动跳过功能失效

### DISCUSSION THREAD

**Discussion by zhanghaotianle** [gkd-kit/gkd#72] [Q&A]
设备:X5 鸿蒙4.0

运行一段时间之后，通常是过夜。自动跳过功能失效，可以确认无障碍服务没有被关闭。关闭后重新开启后恢复正常

**Comment by lisonge** [comment]
看起来和这个一样 https://github.com/gkd-kit/gkd/issues/64

下个版本修复
